test/crdsvalidation/kongcertificate/testcases/required_fields.go

```diff
@@ -0,0 +1,49 @@
+package testcases
+
+import (
+	"github.com/samber/lo"
+
+	configurationv1alpha1 "github.com/kong/kubernetes-configuration/api/configuration/v1alpha1"
+)
+
+var requiredFields = testCasesGroup{
```

Comment:
Do you mind adding a positive case here where are the required fields are specified?